packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -105,7 +106,9 @@ class PictureLayer extends Layer {
 
   @override
   void addToScene(ui.SceneBuilder builder, Offset layerOffset) {
-    builder.addPicture(layerOffset, picture);
+    Timeline.timeSync('PictureLayer::addToScene', () {
+      builder.addPicture(layerOffset, picture);
+    });
```

Comment:
(using the sync versions has two advantages. It's minutely faster since you don't have to create a closure or call it, and it makes it easier to reason about how we would eventually remove calls to Timeline in release builds.)